anime-list-master.xml

```diff
@@ -40805,8 +40811,11 @@
   <anime anidbid="14803" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
     <name>Vampire</name>
   </anime>
-  <anime anidbid="14804" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="14804" tvdbid="259675" defaulttvdbseason="3" episodeoffset="" tmdbid="" imdbid="tt10915840">
     <name>Uchuu Senkan Yamato 2205: Aratanaru Tabidachi</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="3">;401-1;402-2;403-3;404-4;405-5;406-6;407-7;408-8;</mapping>
+    </mapping-list>
```

Comment:
Tricky entry. Feels like it is just "wrong" on AniDB
- The main episodes\movies are not present on TVDB at all so mapping to Season 3 by default feels wrong. 
- "Other" episodes mapped as TVDB Specials in the 400+ range is ASS specific. Does this even work since they're not AniDB Season 0 per-se?

Reply:
I'd probably "unmap" the first 2 episodes\movies of AniDB "Season 1" so that they're not mapped to TVDB's s03e01-e02.
Either `episodeoffset=-2` or `<mapping anidbseason="1" tvdbseason="3">;1-0;2-0;</mapping>` should achieve the same result.
Leaving without a `defaulttvdbseason` doesn't seem like a good idea. Seems better to keep it as 3 just for reference and neuter the regular episodes since they don't exist on TVDB

Reply:
You mean unmap the first 2 episodes\movies of AniDB Season 1
with
`<mapping anidbseason="1" tvdbseason="3">;1-0;2-0;</mapping>`

AND still map 
` <mapping anidbseason="0" tvdbseason="3">;401-1;402-2;403-3;404-4;405-5;406-6;407-7;408-8;</mapping>`

keeping this mapping together ?

Reply:
Correct, otherwise both AniDB 1 and AniDB O1 will be mapped to TVDB s03e01 (same for the second episode)

Reply:
done